muspinsim_input/muspinsim_input.xml

```diff
@@ -35,46 +35,49 @@
         <inputs name="inputs" filename="inputs.json" />
     </configfiles>
     <inputs>
-        <param type="text" name="out_file_prefix" label="A prefix for all output files (default muspinsim)" optional="true" value="muspinsim" />
-        <section name="species" expanded="true" title="Species to simulate">
-            <repeat name="spins" title="Species to simulate">
+        <param type="text" name="out_file_prefix" label="Name" help="A name with which to label this configuration" optional="true" value="muspinsim" />
+        <section name="spins" expanded="true" title="Spins" help="Section for specifying a list of the spins to be used in the system ">
+            <repeat name="spins" title="Species' spins" min="1" help="This should include a muon (mu) and one or more electrons (e)">
```

Comment:
You know what, I'll backtrack on my "'to simulate' is redundant" take. In the section below, the nesting of `Spin Interactions -> Interactions to simulate -> <interaction parameters>` works well, and I think we should do the same in this section.
```suggestion
            <repeat name="spins" title="Spins to simulate" min="1" help="This should include a muon (mu) and one or more electrons (e)">
```